libs/designsystem/checkbox/src/checkbox.component.spec.ts

```diff
@@ -202,4 +202,32 @@ describe('CheckboxComponent', () => {
       expect(spectator.component.checkedChange.emit).toHaveBeenCalledWith(false);
     });
   });
+  describe('ControlValueAccessor implementation', () => {
```

Comment:
```suggestion
  describe('implementing ControlValueAccessor interface', () => {
```

Reply:
This is not important, but it's just the description that is being used in other test files, that tests for ControlCalueAccessor implementation :)